.github/workflows/pr-docker-build.yaml

```diff
@@ -15,7 +15,7 @@
 name: PR Docker Build
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
@josephca FYI, the image does not build properly when it uses `pull_request`. It needs to be `pull_request_target`.

Reply:
@schultzp2020 as the title of PR indicates, it's still [WIP] for a POC purpose. The change in this file is not necessary and won't be delivered. :)

Reply:
Removed from the changeset.